src/db/db.c

```diff
@@ -454,7 +478,7 @@ int dt_db_read(dt_db_t *db, const char *filename)
     lno++;
 
     // scan filename:rating|labels:number
```

Comment:
oh, macintosh thinks 32-bit are long? then probably better to use `%"PRIu64"` here so it will work on all platforms.

Reply:
I just followed compiler warnings, I will see what is going on here, in case ok to switch to PRIu64